<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>0</y>
      <width>369</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <item>
        <widget class="QPushButton" name="start_lidar">
         <property name="text">
          <string>Start Lidar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="end_lidar">
         <property name="text">
          <string>End Lidar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="quit_program">
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="voice_control_start">
       <property name="text">
        <string>Start Voice Control</string>
       </property>
      </widget>
     </item>
    <item>
      <widget class="QPushButton" name="voice_control_stop">
       <property name="text">
        <string>Stop Voice Control</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="start_follow">
       <property name="text">
        <string>Follow Me</string>
       </property>
      </widget>
     </item>
      <item>
      <widget class="QPushButton" name="stop_follow">
       <property name="text">
        <string>Stop Follow</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="start_obstacle_avoidance">
         <property name="text">
          <string>obstacle avoidance</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="end_obstacle_avoidance">
         <property name="text">
          <string>End</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
